index.html

```diff
@@ -1,78 +1,107 @@
+                    aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="about.html">À propos</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="projects.html">Mes projets</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contact.html">Contact</a>
+                        </li>
+                    </ul>
+                </div>
             </div>
-        </div>
-    </nav>
+        </nav>
+    </header>
 
-    <div class="container mt-3">
-        <h1>Bienvenue sur ma page Bootstrap</h1>
-        <p>Cette page est créée avec Bootstrap.</p>
-    </div>
-    <div class="container">
+    <main class="container mb-3">
         <div class="row">
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/dog.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
+            <div class="col-12">
+                <h1 class="text-center">Bienvenue sur mon site</h1>
+                <img src="assets/portrait.jpg" class="img-fluid rounded mx-auto mb-2 d-block h-50" alt="Responsive image">
+                <div class="container w-75">
+                    <p class="text-justify">Bonjour je suis un étudiant en développment web qui est prévue d'être diplômé en mai 2024. J'ai un AEC en programmation d'analyse</p>
                 </div>
             </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/woman.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+
+        <div class="row d-flex justify-content-center">
```

Comment:
The carousel was a good Idea. Just take care to make the images more visible.

![image](https://user-images.githubusercontent.com/6362750/235277497-816858bc-5674-4749-88f8-46d3bf23baff.png)